ACL command panics when asking for info of a non-existing user

<!-- If you suspect this could be a bug, follow the template. -->

### What version of Dgraph are you using?
v1.1.0-6-g3e9810d6

### Steps to reproduce the issue (command/config used to run Dgraph)
```
dgraph acl -a localhost:9080 info -u user1
```

### Expected behaviour and actual result.
*Expected*: error that user doesn't exist
*Actual*: panic
```
[Decoder]: Using assembly version of decoder

Running transaction with dgraph endpoint: localhost:9180
Current password for groot:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa00798]

goroutine 1 [running]:
github.com/dgraph-io/dgraph/ee/acl.info(0xc000312a50, 0x191e6e0, 0xc0006e29c0)
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/ee/acl/acl.go:668 +0x2f9
github.com/dgraph-io/dgraph/ee/acl.initSubcommands.func4(0xc000324fc0, 0xc0006fa580, 0x0, 0x4)
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/ee/acl/run_ee.go:125 +0x32
github.com/dgraph-io/dgraph/vendor/github.com/spf13/cobra.(*Command).execute(0xc000324fc0, 0xc0006fa4c0, 0x4, 0x4, 0xc000324fc0, 0xc0006fa4c0)
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/vendor/github.com/spf13/cobra/command.go:702 +0x289
github.com/dgraph-io/dgraph/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x22151a0, 0x0, 0x0, 0x0)
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/vendor/github.com/spf13/cobra/command.go:783 +0x2ca
github.com/dgraph-io/dgraph/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/vendor/github.com/spf13/cobra/command.go:736
github.com/dgraph-io/dgraph/dgraph/cmd.Execute()
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/dgraph/cmd/root.go:68 +0x85
main.main()
	/home/aman/gocode/src/github.com/dgraph-io/dgraph/dgraph/main.go:33 +0x99
```

--- a/ee/acl/acl.go
+++ b/ee/acl/acl.go
@@ -663,10 +663,10 @@ func info(conf *viper.Viper) error {
 	}
 
 	dc, cancel, err := getClientWithAdminCtx(conf)
-	defer cancel()
 	if err != nil {
 		return errors.Wrapf(err, "unable to get admin context")
 	}
+	defer cancel()
 
 	ctx, ctxCancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer ctxCancel()

